doc/_static/switcher.json

```diff
@@ -0,0 +1,105 @@
+[
+    {
+        "name": "3.5.1 (stable)",
+        "version": "stable"
+    },
+    {
+        "name": "3.6 (devdocs)",
+        "version": "devdocs"
```

Comment:
Do we actually want this showing up in the user-facing version options? Perhaps it should go at the bottom?

Reply:
We link it prominently on the front page, so I think it should be at the top